docs/getting-started/shipping-some-logs.md

```diff
@@ -2,56 +2,142 @@
+         }, {
+           "_index" : "logstash-2015.01.20",
+           "_type" : "nginx_combined",
+           "_id" : "QgFRZr_dSwCS3z3m7q3-vw",
+           "_score" : 0.5501497,
+           "_source":{"@version":"1","@timestamp":"2015-01-20T16:56:40.000Z","host":"10.0.2.15","@type":"nginx_combined","@message":"<14>1 2015-01-20T16:02:19.514+0000 logsearch-workspace LOGSTASH - - [LOGSTASH@1.4.0 @type=nginx_combined] 66.249.81.235 - - [20/Jan/2015:16:56:40 +0000] \"GET /favicon.ico HTTP/1.1\" 404 142 \"-\" \"Mozilla/5.0 (Windows NT 6.1; rv:6.0) Gecko/20110814 Firefox/6.0 Google favicon\" 0.000","syslog_pri":"14","syslog5424_ver":1,"syslog_program":"LOGSTASH","syslog_message":"66.249.81.235 - - [20/Jan/2015:16:56:40 +0000] \"GET /favicon.ico HTTP/1.1\" 404 142 \"-\" \"Mozilla/5.0 (Windows NT 6.1; rv:6.0) Gecko/20110814 Firefox/6.0 Google favicon\" 0.000","received_at":"2015-01-20 16:02:19 +0000","received_from":"10.0.2.15","tags":["syslog_standard","nginx"],"syslog_severity_code":6,"syslog_facility_code":1,"syslog_facility":"user-level","syslog_severity":"informational","@source.host":"logsearch-workspace","syslog_procid":"-","syslog_msgid":"-","syslog_sd_id":"LOGSTASH@1.4.0","syslog_sd_params":{},"remote_addr":"66.249.81.235","remote_user":"-","time_local":"20/Jan/2015:16:56:40 +0000","request_method":"GET","request_uri":"/favicon.ico","request_httpversion":"1.1","status":404,"body_bytes_sent":142,"http_user_agent":"\"Mozilla/5.0 (Windows NT 6.1; rv:6.0) Gecko/20110814 Firefox/6.0 Google favicon\"","request_time":0,"geoip":{"ip":"66.249.81.235","country_code2":"US","country_code3":"USA","country_name":"United States","continent_code":"NA","latitude":38.0,"longitude":-97.0,"dma_code":0,"area_code":0,"location":[-97.0,38.0]}}
+         } ]
+       }
+     }
+
+However, a much more intuitive and visual way to analyse the logs is using the Kibana webapp deployed with your Logsearch cluster.  
+
+Under the covers this is issuing similar HTTP search queries; but then displaying the results in readable charts and tables.
+
+---
 **Next Topic**:  
-[Customizing Log Parsing](./customizing-log-parsing.md)
+[Creating Kibana Dashboards](./creating-kibana-dashboards.md)
 
 Advanced Topics:  
+[Customizing Log Parsing](./customizing-log-parsing.md)
 [Log Shippers](../resources/log-shippers.md)
```

Comment:
These llinks show up one one line, which makes them difficult to distinguish - I propose an unordered list instead to match the TOC style (though a simple line break should do as well, if that's not desired here).
* note that this issue applies to several documents - I'll try to comment as I pass by, but might miss some of course ...